test/utils/init.js

```diff
@@ -20,6 +20,34 @@ chai.use((chaiAPI, utils) => {
       prettyDOM(document.activeElement),
     );
   });
+
+  chai.Assertion.addProperty('ariaHidden', function elementIsAccessible() {
+    const element = utils.flag(this, 'object');
+
+    // "An element is considered hidden if it, or any of its ancestors are not
+    // rendered or have their aria-hidden attribute value set to true."
+    // -- https://www.w3.org/TR/wai-aria-1.1/#aria-hidden
+    let previousNode = element;
```

Comment:
add as comment: only used for debug output. if the loop stopped `currentNode` would either be `null` (no use), point to `html` (no use) or point to the parent of the element that declared aria-hidden.